src/components/general/mediaTypes/ImagePopup.tsx

```diff
@@ -0,0 +1,174 @@
+/* Import Dependencies */
+import classNames from 'classnames';
+import { Formik, Form, Field } from 'formik';
+import KeycloakService from 'keycloak/Keycloak';
+import { ImageAnnotation } from '@annotorious/react';
+import { Row, Col } from 'react-bootstrap';
+
+/* Import Types */
+import { Dict } from 'global/Types';
+
+/* Import Styles */
+import styles from './mediaTypes.module.scss';
+
+/* Import Icons */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPencil, faTrashCan } from '@fortawesome/free-solid-svg-icons';
+
+
+/* Props Typing */
+interface Props {
+    selectedAnnotation: ImageAnnotation,
+    editAnnotation: ImageAnnotation,
+    tooltipFieldRef: any,
+    anno: any,
+
+    RefreshAnnotations: Function,
+    SubmitAnnotation: Function,
+    SetSelectedAnnotation: Function,
+    SetEditAnnotation: Function,
+    RemoveAnnotation: Function
+};
+
+
+const ImagePopup = (props: Props) => {
+    const { selectedAnnotation, editAnnotation, tooltipFieldRef, anno,
```

Comment:
What does `anno` mean, maybe improve this variable name

Reply:
It basically holds the Annotorious object, used the variable as it is also called 'anno' in the author's code.
Changed it to 'annotorious'